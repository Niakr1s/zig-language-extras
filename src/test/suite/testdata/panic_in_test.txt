Running: zig test src/tcp_echo_test.zig

1/7 test.echo server... thread 679934 panic: access of union field 'send' while field 'accept' is active
/home/ianic/zig/io_uring/loop/src/tcp.zig:218:29: 0x2237f3 in write (test)
        self.completion.args.send.buffer = buffer;
                            ^
/home/ianic/zig/io_uring/loop/src/tcp_echo_test.zig:186:30: 0x25ba5b in writeResolve (test)
            self.writer.write(self.writer_buffer[self.writer_pos..to]);
                             ^
/home/ianic/zig/io_uring/loop/src/tcp.zig:185:28: 0x249473 in complete (test)
                    resolve(ctx, no_bytes);
                           ^
/home/ianic/zig/io_uring/loop/src/completion.zig:112:22: 0x224e43 in completed (test)
        self.complete(self, ose, res, flags);
                     ^
/home/ianic/zig/io_uring/loop/src/loop.zig:108:37: 0x224cdf in flush_completions (test)
                completion.completed(cqe.err(), cqe.res, cqe.flags);
                                    ^
/home/ianic/zig/io_uring/loop/src/loop.zig:72:57: 0x22502b in run (test)
            const completed = try self.flush_completions(0);
                                                        ^
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/Thread.zig:433:13: 0x249b6b in callFn__anon_7160 (test)
            @call(.auto, f, args) catch |err| {
            ^
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/Thread.zig:1210:30: 0x22b293 in entryFn (test)
                return callFn(f, self.fn_args);
                             ^
???:?:?: 0x28b0bb in ??? (???)
???:?:?: 0xffffe6de74ef in ??? (???)
???:?:?: 0xffffe6deef87 in ??? (???)
error: the following test command crashed:
/home/ianic/zig/io_uring/loop/zig-cache/o/dc6f017e6b76bd7baf0ec1139d3ce4d4/test

