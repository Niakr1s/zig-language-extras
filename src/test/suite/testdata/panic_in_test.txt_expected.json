[
  {
    "file": "/home/ianic/zig/io_uring/loop/src/tcp.zig",
    "line": 217,
    "column": 28,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin write (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/tcp_echo_test.zig",
    "line": 185,
    "column": 29,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin writeResolve (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/tcp.zig",
    "line": 184,
    "column": 27,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin complete (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/completion.zig",
    "line": 111,
    "column": 21,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin completed (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/loop.zig",
    "line": 107,
    "column": 36,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin flush_completions (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/loop.zig",
    "line": 71,
    "column": 56,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin run (test)",
    "severity": 0
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/Thread.zig",
    "line": 432,
    "column": 12,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin callFn__anon_7160 (test)",
    "severity": 0
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/Thread.zig",
    "line": 1209,
    "column": 29,
    "message": "panic: access of union field 'send' while field 'accept' is active\nin entryFn (test)",
    "severity": 0
  }
]